lib/ecto/repo.ex

```diff
@@ -1158,6 +1158,9 @@ defmodule Ecto.Repo do
       from(p in Post, where: p.id < 10, update: [set: [title: fragment("upper(?)", ^new_title)]])
       |> MyRepo.update_all([])
 
+      from(p in Post, where: p.id < 10, update: [set: [view_count: p.view_count * 1000]])
+      |> MyRepo.update_all([])
```

Comment:
```suggestion
      from(p in Post, where: p.id < 10, update: [set: [visits: p.visits * 1000]])
      |> MyRepo.update_all([])
```
Suggesting for consistency with other examples for this function